src/pages/Home.tsx

```diff
@@ -19,7 +19,23 @@ function Home() {
 
   return (
     <Container component="main" className={classes.main} maxWidth="lg">
-      {/* <Sequencer /> */}
+      <h1>Drumroll Please</h1>
+      <a href='/loops'>
```

Comment:
this needs to be a react router link otherwise it won't work
https://deploy-preview-15--f1v-drum-machine.netlify.app/